src/hotspot/share/gc/shenandoah/shenandoahFreeSet.cpp

```diff
@@ -1189,23 +1843,41 @@ void ShenandoahFreeSet::log_status() {
   // Dump of the FreeSet details is only enabled if assertions are enabled
   if (LogTarget(Debug, gc, free)::is_enabled()) {
 #define BUFFER_SIZE 80
+    size_t retired_old = 0;
+    size_t retired_old_humongous = 0;
+    size_t retired_young = 0;
+    size_t retired_young_humongous = 0;
     size_t region_size_bytes = ShenandoahHeapRegion::region_size_bytes();
+    size_t retired_young_waste = 0;
+    size_t retired_old_waste = 0;
     size_t consumed_collector = 0;
-    size_t available_collector = 0;
+    size_t consumed_old_collector = 0;
     size_t consumed_mutator = 0;
+    size_t available_old = 0;
```

Comment:
Sonar: `available_old` is not used.  `available_young` is not used.

Reply:
https://bugs.openjdk.org/browse/JDK-8344321